### PR TITLE
[dns_server] Fix for for issue 724

### PR DESCRIPTION
**Pull request by neilmunday**
Fixes issue #724

### CODE REVIEW

File: roles/core/dns_server/templates/reverse.j2 [hunk 1]
```diff
@@ -25,7 +25,7 @@ $INCLUDE "{{ dns_server_named_dir }}/{{ item }}.rr.soa"
   {% endif %}
   {% if host_dict['bmc'] is defined %}
     {% set bmc_args = host_dict['bmc'] %}
-    {% if bmc_args.name is defined and bmc_args.name is not none and bmc_args.ip4 is defined and bmc_args.ip4 is not none and bmc_args.network is defined and bmc_args.network is not none and bmc_args.network == outer_item and bmc_args.ip4.split('.')[0:3]|join('.') == item  %}
+    {% if bmc_args.name is defined and bmc_args.name is not none and bmc_args.network is defined and bmc_args.network is not none and bmc_args.network is defined and bmc_args.network.is_in_dns and bmc_args.ip4 is defined and bmc_args.ip4 is not none and bmc_args.ip4.split('.')[0:3]|join('.') == item  %}
```
Comment (oxedions) [inline roles/core/dns_server/templates/reverse.j2:28]: Considering error in Ubuntu global run, I guess here you miss a check on is_in_dns (if defined probably)

**Review by oxedions — CHANGES_REQUESTED** [review]
See my comment 🙂

File: roles/core/dns_server/templates/reverse.j2 (hunk 1 above)
Comment (oxedions) [reply roles/core/dns_server/templates/reverse.j2:28]: https://github.com/bluebanquise/bluebanquise/runs/6905696083?check_suite_focus=true

Comment (neilmunday) [reply roles/core/dns_server/templates/reverse.j2]: Change committed.

**Comment by oxedions** [comment]
Dear @neilmunday 
Thanks a lot for this impressive PR ! 😊 

Merging into master and dev/2.0

**Pull request merged**